src/CommunityToolkit.Maui/HandlerImplementation/Popup/Popup.macios.cs

```diff
@@ -29,6 +29,11 @@ static PageHandler CreatePageHandler(IPopup virtualView)
 				Content = view
 			};
 
+			if (virtualView.Parent is Element element)
```

Comment:
don't think we need this `if` check... Every parent should be an `Element`

Reply:
I only check now for null (Because the Parent is nullable).

Reply:
I added Conditional Compilation for using `AddLogicalChild` as commented here.

https://github.com/dotnet/maui/issues/15760#issuecomment-1676027428

Reply:
Discovered that for .NET 8 a lot of nullable warnings and errors need to be fixed.

for testing i Ignored them by changing following Directory.Build.props (not in pull request).

 `<WarningsAsErrors>nullable,CS0419,CS1570,CS1571,CS1572,CS1573,CS1574,CS1580,CS1581,CS1584,CS1589,CS1590,CS1592,CS1598,CS1658,CS1734</WarningsAsErrors>`

to

 `<WarningsAsErrors>CS0419,CS1570,CS1571,CS1572,CS1573,CS1574,CS1580,CS1581,CS1584,CS1589,CS1590,CS1592,CS1598,CS1658,CS1734</WarningsAsErrors>`

Reply:
No worries! We won't be targeting .NET 8 until it GAs in November. Thanks for the heads up!

Reply:
@inforithmics, yes it's nullable, so you can set null to it. So you can remove all the if code and just assign the Parent value